.github/workflows/ci-cd.yml

```
@@ -0,0 +1,297 @@
+        uses: docker/login-action@v2
+        if: github.event_name == 'push'
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ github.event.repository.name | lower }}
```

actionlint (1.7.9)

got unexpected character ' ' while lexing || operator, expecting '|'

(expression)